infra/gcp/terraform/k8s-infra-prow-build-trusted/prow-build-trusted/resources/test-pods/test-pods-pvcs.yaml

```diff
@@ -0,0 +1,15 @@
+# Prow PVCs
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: publishing-bot-pvc
+  namespace: test-pods
```

Comment:
q: Is the same namespace in which all the prow test jobs run?

Reply:
Yes